README.md

```diff
@@ -97,3 +95,7 @@ Sample scripts can be found in the `src/main/scripts` directory.
 ## Versioning
 
 Follows [Semantic Versioning](http://semver.org/) guidelines.
+
+## Kudos
+
+Kudos to ICF Next / CID 15 for developing this plugin. We forked this plugin because the maintenance of the plugins seems to have stopped.
```

Comment:
```suggestion
Kudos to [ICF Next](https://github.com/icfnext/aem-groovy-console) and [CID 15](https://github.com/CID15/aem-groovy-console) for developing this plugin. We forked this plugin because the maintenance of the plugins seems to have stopped.
```